lib/Navigation/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+const Navigation = props => (
+  <ul className="project-nav">
```

Comment:
In `NavTabs` we use a `<nav>` tag which I believe is more inline with what this is.

Reply:
Done